### PR TITLE
Add new test cases to increase code coverage

### LINKED ISSUE #328: Add new test cases to increase code coverage

### PR DESCRIPTION
**Pull request by Hikoya**
Resolve #328

### CODE REVIEW

File: src/test/java/foodwhere/model/AddressBookTest.java [hunk 1]
```diff
@@ -87,7 +104,6 @@ public void getStallList_modifyList_throwsUnsupportedOperationException() {
 
     @Test
     public void addStallAddReview_generalTesting_success() {
-        AddressBook addressBook = new AddressBook();
         String testName = "test stall";
```
Comment (clarence-chew) [inline src/test/java/foodwhere/model/AddressBookTest.java:107]: I think this test can also be improved to set the address of the stall and review to be the same (and perhaps checking for the cases where one differs).

**Review by clarence-chew — APPROVED** [review]
Some things that I think can be improved, but still okay to merge.

File: src/test/java/foodwhere/commons/core/VersionTest.java [hunk 2]
```diff
@@ -141,6 +142,19 @@ public void versionComparable_validVersion_equalIsCorrect() {
         assertTrue(one.equals(another));
     }
 
+    @Test
+    public void equals() {
```
[the remark this answers is not in the document]
Comment (Hikoya) [reply src/test/java/foodwhere/commons/core/VersionTest.java]: Fixed in latest commit

File: src/test/java/foodwhere/model/AddressBookTest.java (hunk 1 above)
Comment (Hikoya) [reply src/test/java/foodwhere/model/AddressBookTest.java:108]: Fixed in latest commit

**Review by MinHeinA — APPROVED** [review]
All issues addressed. LGTM!

**Pull request merged**